### PR TITLE
GUM-38:  Add experience (connected to backend)

### PR DESCRIPTION
**Pull request by katyakostikova**

### CODE REVIEW

File: src/screens/add-experience/common.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const defaultAddExperiencePayload = {
-  companyName: '',
+  company: '',
   position: '',
-  startDate: undefined,
+  startDate: new Date(),
```
Comment (yevhenprybytok) [inline src/screens/add-experience/common.ts]: I think we need to use iso 8601 format in the logic

**Review by yevhenprybytok — APPROVED** [review]

File: src/common/types/career/add-career.interface.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+interface IAddCareer {
```
Comment (liza-veis) [inline src/common/types/career/add-career.interface.ts]:
```suggestion
interface IAddCareerPayload {
```

File: src/store/experience/actions.ts [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+
+import { IAddCareer } from 'src/common/types';
+import { careerApi } from 'src/services';
+import { ActionTypes } from './common';
+
+const getCareerExperience = createAsyncThunk(
```
Comment (liza-veis) [inline src/store/experience/actions.ts]: It's better to name async actions via load not get
```suggestion
const loadCareerExperience = createAsyncThunk(
```

File: src/store/experience/common.ts [hunk 4]
```diff
@@ -0,0 +1,6 @@
+enum ActionTypes {
+  ADD_CAREER_EXPERIENCE = 'career/create',
+  GET_CAREER_EXPERIENCE = 'career/get-list',
```
Comment (liza-veis) [inline src/store/experience/common.ts]:
```suggestion
  LOAD_CAREER_EXPERIENCE = 'career/load-list',
```

File: src/store/experience/slice.ts [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+const { reducer } = slice;
+
+const actions = {
+  ...slice.actions,
+  addCareerExperience,
+  getCareerExperience,
+};
+
+export { reducer, actions };
```
Comment (liza-veis) [inline src/store/experience/slice.ts]: We have slightly changed store structure. Now we export from a slice only it's actions and then merge them with thunkActions in index.ts file. Please take a look at the store/auth section

**Review by liza-veis — CHANGES_REQUESTED** [review]

**Review by liza-veis — APPROVED** [review]

**Pull request merged**